waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml

```diff
@@ -2301,4 +2301,16 @@
         </sql>
     </changeSet>
 
+
+    <!-- 212 control rendering of static panels -->
+    <changeSet author="kamransaleem" id="20160805-212-1">
```

Comment:
aah ok, so an input binding on 'waltz-static-panels' directive which
controls all of them?

On Fri, Aug 5, 2016 at 12:39 PM, David Watkins <notifications@github.com>
wrote:

> In waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml
> <https://github.com/khartec/waltz/pull/393#discussion_r73679272>:
>
> > @@ -2301,4 +2301,16 @@
> >          </sql>
> >      </changeSet>
> >
> > +
> > +    <!-- 212 control rendering of static panels -->
> > +    <changeSet author="kamransaleem" id="20160805-212-1">
>
> I don't think this should be a database change. It's up to the designer to
> choose at render time how the panels would best fit in their page.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/393/files/d575188b599eecce89e97ce4c3a457d79562da43#r73679272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC452zOzsdHPMw_mIkyqmdOH0mBfOUWQks5qcyDygaJpZM4JdmM6>
> .
>



-- 


*Kamran Saleem*
t  +44 (0) 7732 078654
e  kamran@thinkincode.co.uk
w thinkincode.co.uk